src/Snap/Internal/Http/Server/Config.hs

```diff
@@ -130,6 +138,15 @@ instance Show ConfigLog where
     show (ConfigFileLog f) = "log to file " ++ show f
     show (ConfigIoLog _)   = "custom logging handler"
 
+------------------------------------------------------------------------------
+-- | This handler may be used (in conjunction with setErrorLogHandler) to write out error logs in a 
+-- custom manner.
+type ErrorLogHandler = ByteString -> IO ByteString
```

Comment:
Move this into ".Types" (see the _accessLog member, we are already using this type without an alias there)